src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -26,8 +42,16 @@ export const StatusBadge: ChakraComponent<
 > = chakra(
   forwardRef<HTMLDivElement, React.PropsWithChildren<StatusBadgeProps>>(
     (props, ref?) => {
-      const { children, className, id, level = "low", ...rest } = props;
-      const styles = useStyleConfig("StatusBadge", { variant: level });
+      const { children, className, id, level, type, ...rest } = props;
+
+      // let finalVariant;
+      // if (!level && !type) finalVariant = "low";
+      // else if (level) finalVariant = level;
+      // else if (type) finalVariant = type;
+
+      const finalVariant = level ? level : type ? type : "low";
```

Comment:
Should "neutral" be the default value?

Reply:
For backward compatibility, I set a default from the legacy values.